modules/microshift-install-rpms-olm.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-microshift-with-olm-from-rpm-package_{context}"]
+== Installing the Operator Lifecycle Manager (OLM) for {microshift-short} from an RPM package
+
+When you first install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can install the OLM on your {microshift-short} instance using a RPM package. 
```

Comment:
```suggestion
When you install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can install the OLM on your {microshift-short} instance using a RPM package. 
```
Just thinking about users who have a bunch of instances of MicroShift. It never automatically installs OLM, no matter how many times they install a MicroShift, right?

Reply:
From what I know yes. Its never automatically installed, it has to be installed manually